crates/api/src/arc.rs

```diff
@@ -0,0 +1,386 @@
+//! Kitsune2 arcs represent a range of hash locations on the DHT.
+//!
+//! An arc can exist in its own right and refer to a range of locations on the DHT.
+//! It can also be used in context, such as an agent. Where it represents the range of locations
+//! that agent is responsible for.
+
+use serde::{Deserialize, Serialize};
+
+/// A basic definition of a storage arc compatible with the concept of
+/// storage and querying of items in a store that fall within that arc.
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, Default)]
+pub enum BasicArc {
```

Comment:
If there is no longer going to be more complicated Arc types, does "BasicArc" still make sense as a name? At the very least, it distinguishes it from std::sync::Arc...

Reply:
`HashArc` ?

Reply:
Renamed to `StorageArc`. That's what the field using this type was already called. Seems like a reasonable name to me